src/io/bed.rs

```diff
@@ -233,21 +233,14 @@ impl<'a> From<&'a Record> for annot::contig::Contig<String, strand::Strand> {
     /// Returns a `Contig` annotation for the BED record.
     ///
     /// ```
-    /// # extern crate bio;
-    /// # extern crate bio_types;
     /// use bio::io::bed;
     /// use bio_types::annot::contig::Contig;
     /// use bio_types::strand::Strand;
-    /// # use std::error::Error;
-    /// # fn try_main() -> Result<(), Box<Error>> {
     /// let example = b"chr1\t5\t5000\tname1\t0.5";
     /// let mut reader = bed::Reader::new(&example[..]);
-    /// let rec = reader.records().next().ok_or("No record available!")??;
+    /// let rec = reader.records().next().expect("Error: No record available!");
```

Comment:
For some reason, the `next()` method on `bed` `records` is wrapped as `Option<csv::Result<>>`:
https://github.com/rust-bio/rust-bio/blob/789c55865cc5acc6828f2aa468e963ba7546e199/src/io/bed.rs#L76-L86

So this needs two levels of unpacking, I think these expects should describe the behaviour:
```suggestion
    /// let rec = reader.records().next().expect("Found no bed record.").expect("Got a csv::Error");
```

Reply:
This should fix the testing failure.